src/test/java/com/google/sps/RankTest.java

```diff
@@ -151,105 +272,163 @@ public static Schedule createSchedule3() {
 
   // Creates Schedule 4
   public static Schedule createSchedule4() {
-    Section lectureSection1 = new Section("Professor A", monWedFri(9, 00, DURATION_90_MINUTES));
+    Section lectureSection1 = new Section("Professor A", createSectionTimes(MON_WED_FRI, 9, 00, DURATION_90_MINUTES));
     Section labSection1 = null;
     ScheduledCourse course1 = new ScheduledCourse("Operating System", "15410", 
         "Computer Science", 15, true, lectureSection1, labSection1);
     
-    Section lectureSection3 = new Section("Professor C", tuesThurs(12, 00, DURATION_90_MINUTES));
+    Section lectureSection3 = new Section("Professor C", createSectionTimes(TUES_THURS, 12, 00, DURATION_90_MINUTES));
     Section labSection3 = null;
     ScheduledCourse course3 = new ScheduledCourse("Algorithms", "15210", 
         "Computer Science", 15, true, lectureSection3, labSection3);
 
     return new Schedule(Arrays.asList(course1, course3));
   }
 
-  // =========================================================================================================
+  // Big Schedule 1
+  public static Schedule createBigSchedule1() {
```

Comment:
The helper functions in this file can be made private. In general you should limit the visibility of your functions/variables to the smallest scope necessary.

Reply:
Sounds good. I will change everything back to `private`